subsys/mgmt/mcumgr/grp/os_mgmt/src/os_mgmt_reboot_bt.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zcbor_common.h>
+#include <zcbor_encode.h>
+#include <pm_config.h>
+#include <fw_info.h>
+#include <zephyr/mgmt/mcumgr/mgmt/callbacks.h>
+#include <zephyr/mgmt/mcumgr/grp/os_mgmt/os_mgmt.h>
+
+static enum mgmt_cb_return reboot_bt_hook(uint32_t event, enum mgmt_cb_return prev_status,
+						int32_t *rc, uint16_t *group, bool *abort_more,
+						void *data, size_t data_size)
+{
+	int err_rc;
+	struct os_mgmt_reset_data *reboot_data =
+						(struct os_mgmt_reset_data *)data;
+
+	if (event != MGMT_EVT_OP_OS_MGMT_RESET || data_size !=
+	    sizeof(*reboot_data)) {
```

Comment:
the line size is 100 chars, fix this because these can go on one line, etc. in whole PR